backend/tournesol/tests/factories/video.py

```diff
@@ -7,6 +8,7 @@
 from core.tests.factories.user import UserFactory
 from tournesol.models import Entity, EntityCriteriaScore, VideoRateLater
 
+logging.getLogger('factory').setLevel(logging.WARN)
```

Comment:
It seems reasonable to disable log messages related to `factory` and `faker`. However it would be cleaner to set these levels directly in the `LOGGING` configuration in "settings.py":

```py
    "loggers": {
        "factory": {
            "level": "WARN"
        },
        "faker": {
            "level": "INFO"
        }
    },
```

Otherwise, we depend on implicit side-effects when this module is loaded, and this change may not be applied when running tests on other files only.